packages/chat-client/src/controllers/engine.ts

```diff
@@ -228,11 +228,31 @@ export class ChatEngine extends IChatEngine {
   };
 
   public invite: IChatEngine["invite"] = async (invite) => {
-    // resolve peer account pubKey X
-
     const { inviteePublicKey, inviterAccount, inviteeAccount, message } =
       ZInvite.parse(invite);
 
+    const alreadyInvited = Boolean(
+      this.client.chatSentInvites
+        .getAll()
+        .find((inv) => inv.inviteeAccount === invite.inviteeAccount)
```

Comment:
Can avoid the additional boolean casting via `.some()` here to get a boolean off the bat.

Reply:
Keep forgetting about `some`, thank you